ui/packages/catalog-ui-search/src/main/webapp/js/ApplicationSetup.js

```diff
@@ -153,4 +153,5 @@ require([
     };
 
     require('js/ApplicationStart');
+    require('codice-icons/target/codice.font');
```

Comment:
Can we move this to be imported here: https://github.com/codice/ddf/blob/master/ui/packages/catalog-ui-search/src/main/webapp/styles/styles.less

Reply:
It isn't a style its the generated javascript file from webpack for the webfont

Reply:
🤔 I think we might want to switch from using the `webfonts-loader` to using it's dependency `webfonts-generator` at some point.  This'll work for now though.